### ISSUE TITLE
take out blob shadow

### ISSUE THREAD

**Issue by mjmartin23** [mjmartin23/ivr-vision#2]

**Comment by mjmartin23** [comment]
adapt.m does this pretty well

![pic](https://github.com/mjmartin23/ivr-vision/raw/master/outimages/adapt.png)

**Comment by mjmartin23** [comment]
I noticed when running mean_shift on different images, the blob seems to be there in all except `simpler/02.jpg`. This may be an issue with the parameters I'm using? But, in case it wasn't, I have made `norm_rgb.m` which uses the method he suggests in the handout (normalizing the rgb values), then I thresholded out the background fairly easily. It's not quite as crisp, and some of the items are lost because they are so similar in ratio to the background, but I haven't gotten any interference from the blob:

![07](https://cloud.githubusercontent.com/assets/8230455/19348393/93eec9ae-9145-11e6-9254-854c36e881ac.png)

![02](https://cloud.githubusercontent.com/assets/8230455/19348416/b25c1ed2-9145-11e6-8599-286555bf221f.png)

![09](https://cloud.githubusercontent.com/assets/8230455/19348440/d51f9c8c-9145-11e6-9bfe-0ac7b586ce7b.png)

It's also much faster than mean shift. I think with some tinkering we could get it pretty close to perfect.
